### DISCUSSION TITLE
Ziglang implementation

### DISCUSSION THREAD

**Discussion by GioPat** [capnproto/capnproto#1893] [Ideas]
Is there any ongoing implementation of Cap'n proto in Zig? I'd be available for contributions.

**Comment by kentonv** [comment]
There were a few discussions about Zig on the old mailing list:

https://groups.google.com/g/capnproto/search?q=zig

Not sure how far they got.

**Reply by GioPat** [reply]
Hello, thanks for sharing this. I will have a look at their repo!
